### DISCUSSION TITLE
Background Image in Mainmenu ?

### DISCUSSION THREAD

**Discussion by ds2k5** [templaza/astroid-framework#901] [General]
Hi,
how can I set a Backgrond Image insted of a Color in the Menu ?

![grafik](https://github.com/user-attachments/assets/3b34a4ac-8150-4d33-95df-976403933b56)

**Comment by Banuk1616** [comment]
You can add a background image for the menu via the Layout Manager.
Best Regards
<img width="835" alt="menu_image" src="https://github.com/user-attachments/assets/a676b549-b4dc-406f-b814-7c49f3a707dc">

**Reply by ds2k5** [reply]
thanks

![grafik](https://github.com/user-attachments/assets/5cb37b72-01be-45c4-912d-a0d645d6705b)

**Comment by Banuk1616** [comment]
<img width="721" alt="menu_image2" src="https://github.com/user-attachments/assets/19f04518-926e-450d-9e97-274bc2793034">

**Reply by ds2k5** [reply]
@Banuk1616 

thanks.... how to find this ?

![grafik](https://github.com/user-attachments/assets/12a90165-c0ed-4f5e-8d02-ebc805e1a7d3)

**Comment by ds2k5** [comment]
How to fix this at footer ? remove the green color and get the background image

![grafik](https://github.com/user-attachments/assets/bd5b4459-c536-4ddd-a8d0-34e22badb6de)

![grafik](https://github.com/user-attachments/assets/ce2b114a-8da7-430a-8f02-5b5bf830c32c)

![grafik](https://github.com/user-attachments/assets/ead2301b-83f7-4a3e-9e70-fb1c5c95de34)

**Comment by ds2k5** [comment]
found it

![grafik](https://github.com/user-attachments/assets/92487ed8-437d-41c6-8686-abf5ad667934)

**Comment by ds2k5** [comment]
thanks @ALL  for your help!

**Comment by ds2k5** [comment]
@Banuk1616 

How to set the Text Color of 

- copyright
- contact 

![grafik](https://github.com/user-attachments/assets/706e9762-acb5-429e-8703-a2bc7523dc59)

**Comment by Banuk1616** [comment]
The font color for a section can be set in the Layout Manager.

<img width="1061" alt="cc4" src="https://github.com/user-attachments/assets/71edb391-77df-4e9a-aca4-aaef06f16001">

<img width="867" alt="cc2" src="https://github.com/user-attachments/assets/6b19b3ea-0f4f-4ec6-91a8-1abec0c56305">

Depending on how you have set up your footer, it may be necessary to set the font color via custom.css.

**Comment by ds2k5** [comment]
thanks a lot
